### PR TITLE
Add Education Component

### PR DESCRIPTION
**Pull request by WebDevRaj**
Screenshot (36)

### CODE REVIEW

File: src/modules/resume/education/metadata.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+const metadata = {
+  id: 'resume-education',
+  name: "Education",
+  description: ""
```
Comment (RisingGeek) [inline src/modules/resume/education/metadata.js:5]: Add description

Comment (WebDevRaj) [reply src/modules/resume/education/metadata.js]: Done

File: src/modules/resume/education/variants/basic/properties.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          },
+          university: {
+            name: 'College/University',
+            type: propertyTypes.STRING,
+            value: 'College/University',
+            required: true
+          },
+          datesFrom: {
+            name: 'Dates From',
+            type: propertyTypes.STRING,
```
Comment (ManojSatishkumar) [inline src/modules/resume/education/variants/basic/properties.js:41]: there is a DATE  propertyType. 
Can you add it ?

File: src/modules/resume/education/variants/basic/properties.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            value: 'Dates From',
+            required: true
+          },
+          datesTo: {
+            name: 'Dates To',
+            type: propertyTypes.STRING,
```
Comment (ManojSatishkumar) [inline src/modules/resume/education/variants/basic/properties.js:47]: same here

**Comment by ManojSatishkumar** [comment]
I should have added this rule earlier. Pardon me. Please check.

https://github.com/ManojSatishkumar/msk-components/blob/master/src/modules/resume/sample/variants/basic/properties.js#L26-L30

File: src/modules/resume/education/variants/basic/properties.js (hunk 3 above)
Comment (ManojSatishkumar) [reply src/modules/resume/education/variants/basic/properties.js:47]: I will change it, merging it now.

**Pull request merged**